test/json/json_generator_test.rb

```diff
@@ -86,6 +86,9 @@ def test_dump_strict
 
     assert_equal '42', dump(42, strict: true)
     assert_equal 'true', dump(true, strict: true)
+
+    assert_equal '"hello"', dump(:hello, strict: true)
+    assert_equal '"World"', dump("World".to_sym, strict: true)
```

Comment:
Probably should test that `to_json` is not called with strict

Reply:
It's somewhat already asserting that because `json/add/symbol` is loaded in the test suite.